### DISCUSSION TITLE
Documentation for Apollo 5 and Angular 16 is missing.

### DISCUSSION THREAD

**Discussion by JacopoSciampi** [the-guild-org/apollo-angular#2082] [General]
Since Apollo 5 supports Angular 16 I was wondering where is the documentation for a Standalone approach.

I'm having a lot of problems tryna figure out how to make your client work this way.

**Comment by PowerKiKi** [comment]
This should get you started:

https://github.com/kamilkisiela/apollo-angular/blob/de053417180ff96098878894c38c774dfd290279/packages/apollo-angular/schematics/install/files/standalone/graphql.provider.ts#L1C1-L21

Feel free to open PR to add or replace documentation for standalone
